test/clearingHouse/ClearingHouse.openPosition.test.ts

```diff
@@ -1351,17 +1362,24 @@ describe("ClearingHouse openPosition", () => {
                     referralCode: ethers.constants.HashZero,
                 })
 
+                // {
+                //     indexPrice: '151.0',
+                //     markPrice: '151.312772670283486817',
+                //     marketPrice: '146.88860454062238813'
+                // }
+
+                // realizedPnL = -0.000001209503724417
```

Comment:
Can you help indicate that this `realizedPnL` is `fundingPayment`? That'd help with understanding!

Reply:
add more comments. fixed in 825413845711cca44dcb0c51a361f7d7c2f70dd6